### ISSUE TITLE
issue with dropout for the inference

### ISSUE THREAD

**Issue by thiravet** [affromero/AUNets#37]
Hi,

it's a nice and interesting work. But I have a question.

It seems the dropout probability stays at 0.5 for different layers (default value) for the inference with DEMO option (and not only for the training) . Is is normal or did I make a mistake?

Thanks in advance for the answer.

**Comment by thiravet** [comment]
sorry i didn't have the use of pytorch , i forgot to apply eval() to have the model in inference mode.
